--- configure.ac.orig	2013-03-24 11:12:29.000000000 +0400
+++ configure.ac	2013-05-20 15:44:49.013709594 +0400
@@ -30,6 +30,7 @@
 AC_PROG_GREP
 AC_PATH_PROGS(PERL, [perl perl5], )
 AC_PATH_PROGS(ENV_CMD, [env printenv], )
+AM_PROG_CC_C_O
 
 # Checks for header files.
 AC_HEADER_STDC
--- ./build/find_apu.m4	2013-03-24 11:12:29.000000000 +0400
+++ ./build/find_apu.m4.new	2013-05-20 15:45:44.224714562 +0400
@@ -58,7 +58,7 @@
     if test "$verbose_output" -eq 1; then AC_MSG_NOTICE(apu CFLAGS: $APU_CFLAGS); fi
     APU_LDFLAGS="`${APU_CONFIG} --libs`"
     if test "$verbose_output" -eq 1; then AC_MSG_NOTICE(apu LDFLAGS: $APU_LDFLAGS); fi
-    APU_LDADD="`${APU_CONFIG} --link-libtool`"
+    APU_LDADD="`${APU_CONFIG} --link-ld`"
     if test "$verbose_output" -eq 1; then AC_MSG_NOTICE(apu LDADD: $APU_LDADD); fi
 else
     AC_MSG_RESULT([no])
--- ./build/find_apr.m4	2013-03-24 11:12:29.000000000 +0400
+++ ./build/find_apr.m4.new	2013-05-20 15:45:29.970713435 +0400
@@ -61,7 +61,7 @@
     if test "$verbose_output" -eq 1; then AC_MSG_NOTICE(apr CPPFLAGS: $APR_CPPFLAGS); fi
     APR_LDFLAGS="`${APR_CONFIG} --libs`"
     if test "$verbose_output" -eq 1; then AC_MSG_NOTICE(apr LDFLAGS: $APR_LDFLAGS); fi
-    APR_LDADD="`${APR_CONFIG} --link-libtool`"
+    APR_LDADD="`${APR_CONFIG} --link-ld`"
     if test "$verbose_output" -eq 1; then AC_MSG_NOTICE(apr LDADD: $APR_LDADD); fi
 else
     AC_MSG_RESULT([no])
